apps/ledger-live-mobile/src/components/RootNavigator/EarnLiveAppNavigator.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { ScreenName, NavigatorName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+import { useNavigation, useRoute } from "@react-navigation/native";
+import { useSelector } from "react-redux";
+
+import styles from "../../navigation/styles";
+import type { EarnLiveAppNavigatorParamList } from "./types/EarnLiveAppNavigator";
+import type { StackNavigatorNavigation, StackNavigatorProps } from "./types/helpers";
+import { EarnScreen } from "../../screens/PTX/Earn";
+import { BaseNavigatorStackParamList } from "./types/BaseNavigator";
+import { getAccountIdFromWalletAccountId } from "@ledgerhq/live-common/wallet-api/converters";
+import { accountsSelector } from "../../reducers/accounts";
+
+const Stack = createStackNavigator<EarnLiveAppNavigatorParamList>();
+
+const Earn = (_props: StackNavigatorProps<EarnLiveAppNavigatorParamList, ScreenName.Earn>) => {
+  // Earn dashboard feature flag
+  // TODO: update to use specific mobile feature flag
```

Comment:
I think we need to get this mobile ff in before we can merge. Is Xaviar on this?

Reply:
might be good to add one on dev/staging first https://console.firebase.google.com/u/1/project/ledger-live-development/config

Reply:
Xavier can config ptxEarn to work different in android and ios platform. We do not need to create a new one.

Reply:
ah good to know! Thanks Sara!